### PR TITLE
chore: Add new icons and thumbnails to WDS widgets

### LINKED ISSUE #30901: Change icons for WDS widgets in entity and widget explorers

### PR DESCRIPTION
**Pull request by jsartisan**
Fixes #30901 

## Summary by CodeRabbit

- **New Features**
	- Enhanced feature flag checks to always enable specific features without checking flag values.
	- Introduced dynamic sizing for widget icons and thumbnails in the editor, improving visual consistency.
	- Implemented new styling for text input elements to better adapt to various container sizes.
	- Added unique identifiers to improve testability of the widget sidebar.
	- Introduced new SVG properties for widget thumbnails and icons, enriching widget customization options.

- **Refactor**
	- Adjusted gap spacing in the widget sidebar for a more compact layout.

- **Style**
	- Updated text input styling to use flexible width for improved layout responsiveness.

### CODE REVIEW

File: app/client/packages/design-system/widgets/src/styles/src/text-input.module.css [hunk 1]
```diff
@@ -18,7 +18,7 @@
     flex-grow: 1;
     color: var(--color-fg);
     text-overflow: ellipsis;
-    width: 100%;
+    flex-grow: 1;
```
Comment (jsartisan) [inline app/client/packages/design-system/widgets/src/styles/src/text-input.module.css:21]: This was done to fix regressed bug in input widget where prefix node was not able to take width required.

File: app/client/src/pages/Editor/WidgetCard.tsx [hunk 2]
```diff
@@ -46,6 +47,24 @@ export const Wrapper = styled.div`
     font-family: ${(props) => props.theme.fonts.text};
     font-size: ${(props) => props.theme.fontSizes[7]}px;
   }
+
+  ${(props) =>
+    props.isThumbnail &&
```
Comment (jsartisan) [inline app/client/src/pages/Editor/WidgetCard.tsx:52]: here adding the conditional styles only for thumbnail cards

File: app/client/cypress/locators/WidgetLocators.ts [hunk 3]
```diff
@@ -171,7 +171,7 @@ export const buttongroupwidgetlocators = {
   buttongroup: ".t--buttongroup-widget",
   buttonSettingInPropPane: ".t--property-control-buttons .t--edit-column-btn",
   menuSettingInPropPane: ".t--edit-column-btn",
-  newButton: "//*[text()='Add button']",
+  newButton: "//*[text()='Add new button']",
```
Comment (KelvinOm) [inline app/client/cypress/locators/WidgetLocators.ts:174]: @jsartisan  Seems like wrong conflict resolution. Please check it out.

**Review by KelvinOm — DISMISSED** [review]

**Review by KelvinOm — APPROVED** [review]

**Review by jsartisan — COMMENTED** [review]
LGTM!

**Pull request merged**